doozerlib/runtime.py

```diff
@@ -525,14 +557,14 @@ def resolve_image(self, distgit_name, required=True):
         if distgit_name not in self.image_map:
             if not required:
                 return None
-            raise IOError("Unable to find image metadata in group / included images: %s" % distgit_name)
+            raise DoozerFatalError("Unable to find image metadata in group / included images: %s" % distgit_name)
```

Comment:
hooray for converting nondescript IOError to real exception!

Reply:
:D